src/components/markdown.stories.tsx

```diff
@@ -233,6 +235,9 @@ export const Book: StoryObj<typeof Markdown> = {
   args: {
     children: book,
   },
+  play: async ({ canvasElement }) => {
+    await expect(await within(canvasElement).findAllByTestId("book-cover")).toBeTruthy()
```

Comment:
Could we use the `findAllByText` query here instead of using a `data-testid`? I tend to avoid adding `data-testid` attributes if there's a way to query the document semantically.

```suggestion
    await expect(await within(canvasElement).findAllByText(/book cover/i).toBeTruthy()
```